src/components/Events/index.jsx

```diff
@@ -1 +1 @@
-
+export { default as UpcomingEvents } from "./UpcomingEvents";
```

Comment:
This is not necessary, just move all the content in the UpcomingEvents.jsx file into the index file and import it